### PR TITLE
21 addbackgroundimages

### PR DESCRIPTION
**Pull request by DeborahLacerda**

### CODE REVIEW

File: pages/index.jsx [hunk 1]
```diff
@@ -15,17 +16,25 @@ export default function Home() {
             <AppLayout>
                 <Placeholder title='Hero' />
                 <Placeholder id='about' title='About Us' />
-                <Banner
-                    title={labels.ourMission.title}
-                    description={labels.ourMission.description}
-                />
-                <Banner title={labels.ourVision.title} description={labels.ourVision.description} />
+                <div className={styles.logoOnmission}>
```
Comment (marceloglacial) [inline pages/index.jsx:19]: This is a background image, should be set by CSS to the `<main>` tag

File: styles/background.module.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.logoOnJoinUs {
+    background-image: url(/britologohalf.png);
```
Comment (marceloglacial) [inline styles/background.module.css:2]: You don't need 2 different images. set the dimensions using CSS

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: pages/index.jsx (hunk 1 above)
Comment (DeborahLacerda) [reply pages/index.jsx:19]: I put it in the Banner.module.css. Is that what you meant by main?

File: styles/background.module.css (hunk 2 above)
Comment (DeborahLacerda) [reply styles/background.module.css]: done

File: pages/index.jsx (hunk 1 above)
Comment (marceloglacial) [reply pages/index.jsx:19]: This one :) https://github.com/brito-ca/brito-website/blob/eea08c6fdf30a2539faa7f62361251a21a04326c/components/AppLayout/AppLayout.jsx#L11

Comment (DeborahLacerda) [reply pages/index.jsx:26]: ok, thanks. Done.

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**